README.md

```diff
@@ -1,11 +1,13 @@
 [![CircleCI](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master.svg?style=svg)](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master)
 
-# Ethereum UniversalLoginSDK
+# Ethereum Universal Login
 
-UniversalLogin is a design pattern for storing funds and connecting to Ethereum applications, aiming to simplify new users on-boarding.
+Universal Login is a design pattern for storing funds and connecting to Ethereum applications, aiming to simplify new users on-boarding.
 
 This repository is a monorepo including sdk, relayer, smart contracts and example. Each public sub-package is independently published to NPM.
 
+Learn [documentation](https://universalloginsdk.readthedocs.io/en/latest/index.html).
```

Comment:
## Documentation
Documentation is availble at [universalloginsdk.readthedocs.io]()